docs/rules/no-duplicate-imports.md

```diff
@@ -2,6 +2,8 @@
 
 An ES6/ES2015 import can be spread over multiple lines, but this takes up unneeded whitespace. This rules validates that all imports from a single module exists in a single import statement.
 
+Having all imports from the same modules come from the same `import` statement can make the code more clear, and decrease the mental overhead with regards to what modules are imported.
```

Comment:
Better as...

> Using a single import statement per module will make code much clearer; you can see everything being imported from that module on one line, rather than it being spread out over several lines.

?